process-services/latest/config/index.md

```diff
@@ -115,7 +115,8 @@ cors.support.credentials=truecors.preflight.maxage=10
 
 |Property|Description|
 |--------|-----------|
-|cors.allowed.origins|Specifies the hosts allowed in cross origin requests. By default, the value is set to `*`, which permits clients hosted on any server to access the resources. Alternatively, you can specify a host, for example, `http://www.example.org:8080`, which will only allow requests from this host.<br><br>Multiple entries or wildcards are not allowed for this setting.<br><br>In general, it is recommended to restrict `{{allowedOrigins}}` to only allow origins within your organization to make requests.|
+|cors.allowed.origins|Specifies the hosts allowed in cross origin requests. You must specify a host, for example, `http://www.example.org:8080`, which will only allow requests from that host.<br><br>Multiple entries or wildcards are not allowed for this setting.<br><br>In general, it is recommended to restrict `{{allowedOrigins}}` to only allow origins within your organization to make requests.|
+|cors.allowed.origin.patterns|Specifies the hosts allowed in cross origin requests. You can use `*`. You can also specify a host, for example, `http://www.example.org:8080`, which will only allow requests from that host.<br><br>In general, it is recommended to restrict `{{allowedOrigins}}` to only allow origins within your organization to make requests.|
```

Comment:
For `cors.allowed.origin.patterns` I would recommend following the Spring framework description:

Alternative to `cors.allowed.origins` that supports more flexible origins patterns with "*" anywhere in the host name in addition to port lists. Examples:

see: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/cors/CorsConfiguration.html#setAllowedOriginPatterns-java.util.List-

Reply:
Ok fixed this up cheers :-)